panel.go

```diff
@@ -566,7 +566,7 @@ type Target struct {
 	CrossSeriesReducer string                    `json:"crossSeriesReducer,omitempty"`
 	PerSeriesAligner   string                    `json:"perSeriesAligner,omitempty"`
 	ValueType          string                    `json:"valueType,omitempty"`
-	GroupBys           []string                  `json:"groupBys,omitempty"`
+	GroupBy            []string                  `json:"groupBys,omitempty"`
```

Comment:
Doesn't this need to be:
```suggestion
	GroupBy            []string                  `json:"groupBy,omitempty"`
```
? At least that's what you say in the PR description if I understood you correctly :thinking:

Reply:
yes, that's exactly it. I knew I missed a spot somewhere. I've addressed that now.